lib/portlayer/event/vsphere/manager.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vsphere
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"reflect"
+
+	"github.com/vmware/vic/lib/portlayer/event"
+	"github.com/vmware/vic/pkg/vsphere/session"
+
+	vmwEvents "github.com/vmware/govmomi/event"
+	"github.com/vmware/govmomi/vim25/types"
+
+	"golang.org/x/net/context"
+)
+
+type vSphereManager struct {
+	vmwManager *vmwEvents.Manager
+	callbacks  map[string]func(event.Event, *session.Session)
+	blacklist  map[types.ManagedObjectReference]string
+	objects    map[string]types.ManagedObjectReference
+	sess       *session.Session
```

Comment:
ok - so I THOUGHT because it was the package name that's why the other areas used sess instead of session, but apparently that's not the case. A package alias wan't required, so I've changed it to session